### PR TITLE
#161967114 upgrade django app

### PR DESCRIPTION
**Pull request by denomash**
**What does this PR do?**
Upgrades app
<hr>

**Description of Task to be completed?**
Upgrade Authors Haven Django app from Django 1.10 to Django 2.1
<hr>

**Any background context you want to provide?**
Django 2.x is the latest release of Django with full support of Python 3.x series.
<hr>

#### What are the relevant pivotal tracker stories?
[#161967114](https://www.pivotaltracker.com/story/show/161967114/comments/196893277)
<hr>

**Checklist**
 - My code follows the style guidelines of this project
 - I have performed a self-review of my own code
 - I have commented my code, particularly in hard-to-understand areas
 - I have made corresponding changes to the documentation
 - My changes generate no new warnings

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,7 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+# IDE files
+.vscode/
+.idea/
```
Comment (Bryan-Cee) [inline .gitignore]: Just realized this line was repeated, it's already in line 26

Comment (denomash) [reply .gitignore]: Nice observation Brian-Cee. On it

**Comment by Darius-Ndubi** [comment]
On the PR under checklist, You can use a `-` before each line for purposes of easy reading.

**Comment by denomash** [comment]
> On the PR under checklist, You can user - before each line for purposes of easy reading.

nice idea Darius-Ndubi

**Review by Bryan-Cee — COMMENTED** [review]
The changes you've made are good. Just edit the commit message as well since you removed `.idea` from git ignore.

**Review by jillo-abdullahi — CHANGES_REQUESTED** [review]
@denomash `pytz==2018.7` is Django dependency for time zone support. It should be present on the `requirements.txt` file.

**Comment by denomash** [comment]
@jillo-abdullahi, pytz==2018.7 installs automatically to your virtual environment after running `pip install -r requirements.txt`

**Comment by bettblake08** [comment]
@denomash Your PR is all good.

**Comment by denomash** [comment]
@Bryan-Cee edited the commit message. Its all good

**Review by jillo-abdullahi — APPROVED** [review]

**Review by Bryan-Cee — CHANGES_REQUESTED** [review]
Make sure your PR has only one commit.

**Review by Bryan-Cee — APPROVED** [review]

**Review by mutoromike — CHANGES_REQUESTED** [review]
Check why your build is failing on Travis.

**Review by bettblake08 — COMMENTED** [review]
The PR is all good

**Review by mutoromike — APPROVED** [review]

**Pull request merged**